### ISSUE TITLE
Floating Hair/Beard Issues

### ISSUE THREAD

**Issue by yarrick5** [ForgottenGlory/Living-Skyrim-3#630]
**If you are reporting a crash to desktop, please attach your NET Script Framework crash log. This can be found in `[LS Install folder]\mods\Files Generated during playsessions launched with SKSE\NetScriptFramework\Crash`

If possible, please also attach a copy of your most recent save before the issue occurred.**

Please check back on your bug request periodically. I will ask for more details sometimes and if they are not provided I will eventually close the bug report.

**LS Version**
3.6.0.2

**Describe the bug**
During character creation, a number of hair styles are either floating above the character's head or are transparent. Also, several beard styles seem to be having issues, with numerous patterns/"spots" appearing through the beard mesh.

**To Reproduce**
Steps to reproduce the behavior: Occurs during character creation every time.

**Expected behavior**
A clear and concise description of what you expected to happen: Hair and beards should fit closer to character's facial/head structure. 

**Screenshots**

If your report pertains to a specific NPC, object, or item, please open the console using the tilde (~) key and then click on that NPC, object or item. Take a screenshot of this and include that in this report. You will know if you have the correct thing selected if it is highlighted white after clicking on it.

**Additional context**
By no means experienced in modding, I've put together personal lists years ago and understand a bit about mod incompatibilities and the game's file structure, but am not an expert by any means. I appreciate the work that's obviously gone into this, and any input would be great.
![Skyrim Special Edition 1_10_2022 10_37_27 AM](https://user-images.githubusercontent.com/97435339/148739515-52993d9a-2f3c-4065-bfd7-cbd55becc5f8.png)
![Installation Instructions _ ForgottenGlory's Modlists — Mozilla Firefox 1_10_2022 10_47_51 AM](https://user-images.githubusercontent.com/97435339/148739576-9836bbcd-af07-46b6-8cb3-165a638e9ed0.png)

**Comment by celadoneiron** [comment]
You need to choose the High Poly Head headpart _and_ High Poly Head beard, or the vanilla versions of both, you can't mix and match. Some hairstyles float on some heads because they're made for different races but have been unlocked, or because they fit better with either the vanilla or the high poly head. It's unfortunate, but with the number of hairstyles in the list it's usually possible to find something that fits regardless, and if you're really adventurous you can try sculpting the hair in the sculpt tab to get it to fit, too.

If you still get the beard/face clipping issue when you've got the part types matched (you can tell because when you move the slider along, a label will pop up in the bottom right hand corner with a name, and if it starts with something like "00KLH" then it's the High Poly Head version), let us know in a comment and I'll reopen the issue.

**Comment by yarrick5** [comment]
Hi,
Thank you very much for the answer. Frankly, I was exhausted after work
when installing and did not properly read all of the instructions
correctly. Thank you for taking the time to explain.
I've been more thorough now in re-reading and setting things up, so I hope
that I'm not making the same mistake again on a separate issue, but I'm
having a problem involving object pop-in. I've started a new game and have
selected the "Regular at an Inn" - "Dead Man's Drink" start to test things
before starting a more serious character, and on walking around town,
especially near the main town gate, there are a number of buildings and
objects, many quite close to my point of view, that are popping in. This
seems to be determined to some degree also by the angle at which I'm
viewing them.
Any help again would be very much appreciated, and I apologize if I'm
missing something obvious here.

**Comment by yarrick5** [comment]
Thank you very much for the prompt answer. I was exhausted after work and frankly did not properly read the instructions properly, and I really appreciate you taking the time to explain this to me despite it being carelessness on my part.
